week-2-practise/level-2/6-map.js

```diff
@@ -5,10 +5,14 @@
 
 var percentages = [1, 23, 92, 18];
 
+function percentageNumber(num) {
```

Comment:
Nice! But remember that function names should include a `verb`, so an action word - how would you rename this function then?